lib/python/rosie/browser/main.py

```diff
@@ -33,8 +41,11 @@
 
 import pygtk
 pygtk.require("2.0")
+try:
```

Comment:
According to the traceback on the line in the `except` section the `try` section is the preferred format.